### DISCUSSION TITLE
Change IP address and port for Modbus Device

### DISCUSSION THREAD

**Discussion by TomasAdvance** [frangoteam/FUXA#1061] [General]
Hi Fuxa team,
Is any possibility to scrip change IP and port for modbus Device?
If yes may I ask you for example?
If not, will be nice future for custom setting.
BR

**Comment by unocelli** [comment]
Hi, I think it would be more adaptable and safer to be able to activate and deactivate devices via scripts

**Comment by TomasAdvance** [comment]
Hi,
sure, enabling and disabling devices via scrip is simple and easy and I know how to do it. But I would need the ability to change device parameters via scrip (e.g. IP address, port, etc.)

**Reply by unocelli** [reply]
Hi, we would need a get/set device property, we currently have other priorities but we would be very happy to add it (PR :wink:)

**Comment by unocelli** [comment]
Hi, the feature is now available.

**Comment by rvbatista** [comment]
Great @unocelli this feature is great for smart MCCs that have lots of the same device.
